Change of default for allow_vendor_change (#623)
rpm-software-management/dnf5: doc/changes.rst

Changes of configuration:
=========================

> Defauilt of `allow_vendor_change` configuration option changed to `false`

Reviewer: typo - defauilt -> default

-------------------------------------------------------------------------

> The new default value improves behavior for users that use third party repository (Copr). If they install a package from
> alternative provider, DNF will continue to consume updates from the same vendor. Additionally it enables to ship
> alternative streams using a different vendor.
> 

Reviewer: ```suggestion
The new default value improves the behavior for users of third-party repositories (e.g. Copr). If they install a package from
an alternative provider, DNF will continue to consume updates from the same vendor. Additionally it enables the shipping of
alternative streams using a different vendor.

```
